server/src/omtDocumentInformationProvider.ts

```diff
@@ -186,15 +198,26 @@ function findUsagesInLine(declaredObjects: string[], lineNumber: number, line: s
 /**
  * Create a DocumentLink linking to another document or to be resolved later.
  * @param line the line number within the document where the link should be
- * @param start the start position on the line
- * @param length the length of the text representing the link
+ * @param omtLocalObject the object where the link will be placed
  * @param uri the path to the document. leave undefined if the link will be resolved using the resolveDocumentLink request
  * @param data the data passed with the link when resolving it with the resolveDocumentLink request
  */
-function createDocumentLink(line: number, start: number, length: number, uri: string | undefined, data?: unknown): DocumentLink {
+function createDocumentLink(line: number, omtLocalObject: OmtLocalObject, uri: string | undefined, data?: unknown): DocumentLink {
+    const start = omtLocalObject.range.start.character;
+    const length = omtLocalObject.name.length;
     const from = Position.create(line, start);
     const to = Position.create(line, start + length);
-    return DocumentLink.create(Range.create(from, to), uri, data);
+    const declaredImportModule = getDiMatch(omtLocalObject.name);
+    if (declaredImportModule) {
+        return DocumentLink.create(Range.create(from, to), undefined, {
+            declaredImport: {
+                module: declaredImportModule
+            }
+        });
+    }
```

Comment:
The `else` should go on the same line as the if's closing brace

Reply:
Which formatter do you use? I'm using the default in VSCode, which doesn't care about where the else is placed. I will change this line.

Reply:
The repo had ESLint configuration, but I'm not sure which rules are enforced. Do you have the ESLint plugin in VSCode?